lib/qesapdeployment.pm

```diff
@@ -540,7 +540,7 @@ sub qesap_execute_conditional_retry {
     $args{retries} //= 1;
 
     my @ret = qesap_execute(cmd => $args{cmd},
-        verbose => $args{verbose},
+        ($args{cmd} eq 'ansible' ? () : (verbose => '--verbose')),
```

Comment:
strange syntax in the middle of conditional code. How it works?

Reply:
If $args{cmd} is equal to 'ansible', the condition is true, so () is returned. This is an empty list.
If $args{cmd} is not equal to 'ansible', the condition is false, so (verbose => '--verbose') is returned. That means verbosity will be activate with the '-vvv'

Reply:
If `$args{cmd}` is the same as the `ansible` string, then return an empty list; otherwise return a list with 2 elements: `verbose` and `--verbose`.

Reply:
Have to add that this works because the ternary operator is limiting a list of 0 or 2 elements from being added to `%args` ... if it were an odd number, it could create trouble.